bulb.py

```diff
@@ -0,0 +1,59 @@
+
+'''Class that contains the set of rules around sensors'''
+class Bulb(object):
+
+    def __init__(self,  name, lightSensor, presenceSensors, homeSensor, wastage_function):
+        self.name = name
+        self.light_sensor = lightSensor
+        self.presence_sensors = presenceSensors
+        self.home_sensor = homeSensor
+        self.wastage_function = wastage_function
+
+
+    def isWasted(self, *args):
+        return self.wastage_function(self,*args)
+
+
+# Actual function implementation for the given lights
+
+def kitchenLampWasted(self):
+    # assumes self.presence_sensors = [desk_sensor, bed_sensor]
+    if not self.light_sensor.isOn():
+        return False, 'OFF'
```

Comment:
The actual string values could come from a single place/class.. Something like an Enum. I don't know how enums are done in python, but if we ever need to refactor or anything, then having code littered with actual literal string is a bad smell :)